OutOfSchool/OutOfSchool.Common/PermissionModule/PermissionManagement/PermissionsSeeder.cs

```diff
@@ -17,7 +17,7 @@ public static class PermissionsSeeder
             Permissions.TeacherAddNew, Permissions.TeacherEdit, Permissions.TeacherRemove, Permissions.TeacherRead,
             Permissions.UserRead, Permissions.UserEdit,
             Permissions.WorkshopEdit, Permissions.WorkshopRemove, Permissions.WorkshopAddNew,
-            Permissions.MinistryAdmins, Permissions.MinistryAdminAddNew, Permissions.MinistryAdminRemove,
+            Permissions.MinistryAdmins, Permissions.MinistryAdminAddNew, Permissions.MinistryAdminRemove, Permissions.MinistryAdminEdit, Permissions.MinistryAdminRemove,
```

Comment:
`Permissions.MinistryAdminRemove` has duplicate. Please clean excessive code

Reply:
Fixed in new PR #747